stage1/init/kvm/network.go

```diff
@@ -158,7 +158,9 @@ func GenerateNetworkInterfaceUnits(unitsPath string, netDescriptions []netDescri
 			return errwrap.Wrap(fmt.Errorf("failed to create network unit file %q", unitName), err)
 		}
 
-		rlog.Printf("network unit created: %q in %q (iface=%q, addr=%q)", unitName, unitsPath, ifName, address)
+		if debug {
```

Comment:
It doesn't help.
When I used rlog.SetDebug(debug) before rlog.Printf, message was logged every time - no difference when --debug was give.